source/jormungandr/jormungandr/instance.py

```diff
@@ -559,13 +559,11 @@ def _send_and_receive(
         deadline = datetime.utcnow() + timedelta(milliseconds=timeout)
         request.deadline = deadline.strftime('%Y%m%dT%H%M%S,%f')
         with self.socket(self.context) as socket:
-            try:
-                request.request_id = flask.request.id
```

Comment:
Does this change mean that the flask request id was useless?

Reply:
No it wasn't, I added it back.